app/src/main/java/edu/mit/mitmobile2/dining/activities/DinningHouseActivity.java

```diff
@@ -0,0 +1,104 @@
+package edu.mit.mitmobile2.dining.activities;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.v4.view.ViewPager;
+import android.view.Menu;
+import android.view.MenuItem;
+import android.view.View;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import com.squareup.picasso.Picasso;
+
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+
+import butterknife.ButterKnife;
+import butterknife.InjectView;
+import butterknife.OnClick;
+import edu.mit.mitmobile2.Constants;
+import edu.mit.mitmobile2.MITActivity;
+import edu.mit.mitmobile2.R;
+import edu.mit.mitmobile2.dining.adapters.HouseMenuPagerAdapter;
+import edu.mit.mitmobile2.dining.model.MITDiningHouseVenue;
+
+public class DinningHouseActivity extends MITActivity {
```

Comment:
Only extend MITActivity if you need location, which I don't think is necessary here.

Otherwise extend AppCompatActivity